archive.py

```diff
@@ -376,13 +377,19 @@ def write_last_updated(t):
     f.write('<hr><p>Last updated: {} UTC</p>'.format(t))
     f.close()
 
+def write_css():
+    outfile = open_outfile(md_root, 'style.css', 'w+')
+    outfile.write(".msg { margin-left: 2em; }\n")
+    outfile.close()
```

Comment:
I think a cleaner system would be to just have a `style.css` in this project and have this code copy it into place (or maybe just link to it).  That way, folks editing CSS just need to edit a single CSS file, not using this weird format, to get the styling they want.

Reply:
Updated to be having a `style.css` that is then copied over.